dts/arm/nordic/nrf51822.dtsi

```diff
@@ -140,6 +141,14 @@
 			label = "SPI_1";
 		};
 
+		temp: temp@4000c000 {
+			compatible = "nordic,nrf-temp";
+			reg = <0x4000c000 0x1000>;
+			interrupts = <12 1>;
+			status = "ok";
+			label = "TEMP";
```

Comment:
"TEMP_0", to stay consistent with the default value of `CONFIG_TEMP_NRF5_NAME`.

Reply:
Changed to TEMP_0.